pkgs/development/python-modules/pyinstaller-hooks-contrib/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pyinstaller-hooks-contrib";
+  version = "2024.7";
+
+  format = "setuptools";
+
+  src = fetchPypi {
+    pname = "pyinstaller_hooks_contrib";
+    inherit version;
+    hash = "sha256-/V833Pmb7OGE5AZCr4i+Fqm4lhPsuViovRE2Y0/J+sU=";
+  };
+
+  # There are tests for every hook, which means that
+  # new updates are going to require changes to test inputs
+  # and building tests creates a very big closure.
+  doCheck = false;
+
+  meta = {
+    description = "Community maintained hooks for PyInstaller";
+    longDescription = ''
+      A "hook" file extends PyInstaller to adapt it to the special needs and methods used by a Python package.
+      The word "hook" is used for two kinds of files. A runtime hook helps the bootloader to launch an app,
+      setting up the environment. A package hook (there are several types of those) tells PyInstaller
+      what to include in the final app - such as the data files and (hidden) imports mentioned above.
+      This repository is a collection of hooks for many packages, and allows PyInstaller to work with these packages seamlessly.
+    '';
+    homepage = "https://github.com/pyinstaller/pyinstaller-hooks-contrib";
+    downloadPage = "https://pypi.org/project/pyinstaller-hooks-contrib";
+    changelog = "https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/master/CHANGELOG.rst";
```

Comment:
```suggestion
    changelog = "https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/v${version}/CHANGELOG.rst";

```

Reply:
The reason I set it to master is because they always update this file after they set the tag (e.g. see https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/v2024.7/CHANGELOG.rst). Would it be better if I added a comment, explaining why it is set as it is?

Reply:
Probably file an issue upstream regarding this and link that issue here. Even if its not fixed, an issue seems like a better reference, IMO.